### PR TITLE
Config and landing page updates

### PR DESCRIPTION
**Pull request by Patrick-Erichsen**
- Add site description
- Move blockchain specific info to the `About` page
- Remove external link to ConsenSource app

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 baseURL = "https://target.github.io/consensource-docs/"
 title = "ConsenSource Docs"
+description = "ConsenSource brings transparency and trust to the sourcing industry"
```
Comment (adeebahmed) [inline config.toml]: Certification transparency and authenticity to empower sustainable sourcing

File: config.toml [hunk 2]
```diff
@@ -51,10 +52,10 @@ anchor = "smart"
...
 
 # Top-level navigation (horizontal)
-[[menu.main]]
-    name = "ConsenSource.org"
-    weight = 1
-    url = "https://35.201.118.216"  
+# [[menu.main]]
+#     name = "ConsenSource.org"
+#     weight = 1
+#     url = "https://35.201.118.216"
```
Comment (adeebahmed) [inline config.toml]: remove this area

File: content/_index.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
 	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/target/consensource" target="blank">
 		View the code <i class="fab fa-github ml-2 "></i>
 	</a>
-	<p class="lead mt-5">Certification transparency and authenticity using distributed ledger technology</p>
```
Comment (adeebahmed) [inline content/_index.html]: Certification transparency and authenticity to empower sustainable sourcing

File: content/_index.html [hunk 4]
```diff
@@ -10,7 +10,7 @@
...
+	<p class="lead mt-5">Bringing transparency and trust to the sourcing industry</p>
```
Comment (adeebahmed) [inline content/_index.html]: Do we need this second line?

File: content/about/_index.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ For many brands and retailers, a commom pain point in the sourcing process is th
 In practice, each brand and retailer ends up independently verifying these claims with the ceritfying body that performed the original audit. This process is time-consuming, expensive, and error prone. 
 {{% /blocks/cover %}}
 
-{{% blocks/section type="section" color="dark" %}}
+{{% blocks/section color="white" %}}
```
Comment (adeebahmed) [inline content/about/_index.md]: What is this?

File: config.toml (hunk 1 above)
Comment (Patrick-Erichsen) [reply config.toml]: Certification transparency and authenticity to empower `responsible` sourcing

Comment (adeebahmed) [reply config.toml]: 👍

**Pull request merged**